### ISSUE TITLE
WASAPI init_output_device error in editor and in game running - Godot 4.0.x

### ISSUE THREAD

**Issue by ChidemJean** [godotengine/godot#77563]
### Godot version

4.0.2.stable, 4.0.3.stable

### System information

- OS: Windows 10 Pro
-  Godot Version: 4.0.2.stable.mono, 4.0.3.stable.mono
- Rendering Driver:: Vulkan API 1.3.205 (but in compatibility mode also occurs)
- Rendering method: Forward+, mobile, compatibility
- Graphics Card: NVIDIA GeForce GTX 1050 Ti
- Graphics Card Driver: NVIDIA version 516.94
- CPU: Intel Core i5-7400

### Issue description

In the editor console and when running any project (I started with just a mesh and a camera in this print example) these errors related to the audio device occur. I tried to disable the exclusive device control option as indicated in this issue: https://github.com/godotengine/godot/issues/26841, but it didn't fix it. I also tried to run godot in admin mode and with all programs that use audio closed and it didn't fix it either.
![erro_godot_4](https://github.com/godotengine/godot/assets/10237062/072430e9-489a-4cea-a9fb-c78df74244fc)

### Steps to reproduce

I tried with just one camera inside a 3d node and the error happened when running the game. I opened the engine with the console and the mentioned errors were also occurring even with nothing on the scene.

### Minimal reproduction project

N/A

**Comment by MJacred** [comment]
> I tried to disable the exclusive device control option as indicated in this issue

@ChidemJean: You did disable it like this: https://github.com/godotengine/godot/issues/18732#issuecomment-466753588?  
And it didn't change back again shortly after?

Some note it might be due to the sound card…

---

Related to / Duplicate of:
* https://github.com/godotengine/godot/issues/18732

**Comment by ChidemJean** [comment]
> > I tried to disable the exclusive device control option as indicated in this issue
> 
> @ChidemJean: You did disable it like this: [#18732 (comment)](https://github.com/godotengine/godot/issues/18732#issuecomment-466753588)? And it didn't change back again shortly after?
> 
> Some note it might be due to the sound card…
> 
> Related to / Duplicate of:
> 
> * [AudioDriverWASAPI error while using editor, no audio feature used #18732](https://github.com/godotengine/godot/issues/18732)

Yes, I did as mentioned in that question, it didn't turn itself back on, but the problem persists. What I found most strange is that it only happens in version 4.

![config](https://github.com/godotengine/godot/assets/10237062/599daae8-c0da-4212-84f3-b80fc4ca8523)

**Comment by Succyboi** [comment]
I am also experiencing this issue. Similarly I tried to disable exclusive device control as mentioned in [#26841](https://github.com/godotengine/godot/issues/26841), but the issue persisted.

**Comment by allenwp** [comment]
It is possible that this issue was resolved by #100116, which was included in Godot 4.4 and 4.3.1. I understand it has been a while since you opened this report, but would you be able to test and confirm this @ChidemJean?
